x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/detection_alerts/assignments/assignments.cy.ts

```diff
@@ -77,42 +67,23 @@ describe('Alert user assignment - ESS & Serverless', { tags: ['@ess', '@serverle
     });
 
     it('alert with some assignees in alerts table', () => {
-      const users = [ROLES.detections_admin, ROLES.t1_analyst];
+      const users = [getDefaultUserName()];
       updateAssigneesForFirstAlert(users);
       alertsTableShowsAssigneesForAlert(users);
     });
 
     it(`alert with some assignees in alert's details flyout`, () => {
```

Comment:
Broader question outside the scope of this particular PR: why is this test not part of the one above it? The script appears to be:

1. login
2. create rule, wait for alerts
3. assign alert to user
4. make assertions about assignment

Other than violating the "one assertion per test" rule (which I don't believe is relevant to cypress), is there a reason for not consolidating these? I can imagine that having more tests _seems_ like it would make the suite more robust, but given the amount of work that happens _before_ the assertions (that then gets repeated in every independent test), I believe the opposite is true.

Reply:
Scrolling further it seems as though both of these tests are now redundant with `Updating assignees (single alert) adding new assignees via 'More actions' in alerts table`; that one tests everything the two of these do.

Reply:
Agree! I will walk through tests and will consolidate where it is possible.